src/lib/containers/row_renderers/Dataset.js

```diff
@@ -60,7 +61,7 @@ class Dataset extends React.Component {
                       columns.map(
                           column => {
                               return (
-                                  <div key={uuidv4()} className="col-md-2 col-sm-2">
+                                  <div key={uuidv4()} className="col-md-3 col-sm-3">
```

Comment:
if specifying col-sm-3, that applies to all larger screens (col-md-3 unnecessary).